src/patternfly/components/List/list.scss

```diff
@@ -1,11 +1,26 @@
 .pf-c-list {
+  // list
   --pf-c-list--PaddingLeft: var(--pf-global--spacer--lg);
   --pf-c-list--nested--MarginTop: var(--pf-global--spacer--sm);
   --pf-c-list--nested--MarginLeft: var(--pf-global--spacer--sm);
   --pf-c-list--ul--ListStyle: var(--pf-global--ListStyle);
   --pf-c-list--li--MarginTop: var(--pf-global--spacer--sm);
+
+  // inline
   --pf-c-list--m-inline--li--MarginRight: var(--pf-global--spacer--lg);
 
+  // bordered
+  --pf-c-list--m-bordered--li--PaddingBottom: var(--pf-global--spacer--sm);
```

Comment:
Two questions about how the borders are drawn:

<img width="626" alt="Screen Shot 2021-01-14 at 5 59 41 PM" src="https://user-images.githubusercontent.com/35148959/104663367-7b5f7700-5692-11eb-8c7f-381595c94db4.png">

Since we create borders between items, if you have a nested list in a parent list item, there is no border between the parent item and the first nested item. Is that intended? Are bordered lists valid when nested like this?

Also since we're using `border`, the border doesn't extend underneath the list marker.

@maryshak1996 can you confirm this looks as expected?

Reply:
I am not sure whether we could use borders on nested list, but I am intended not to set border between the parent item and the first nested item. Waiting for designer's check.

Reply:
Hey guys, yes this looks as expected! Dropping the divider line between the parent and first child helps tie parent and child together to remind user of the relationship between the two (I feel like I'm talking about a family reunion right now not UX....)

Reply:
@maryshak1996 Thank you for checking. BTW, do you think whether the border should extend underneath the list marker or not?

Reply:
@DaoDaoNoCode I don't think that we want to extend it out under the markers, no. it looks a bit too heavy if it is.